consumer/src/test/kotlin/streams/StreamsTopicServiceTest.kt

```diff
@@ -0,0 +1,40 @@
+package streams
+
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.neo4j.graphdb.GraphDatabaseService
+import org.neo4j.kernel.internal.GraphDatabaseAPI
+import org.neo4j.test.TestGraphDatabaseFactory
+import streams.kafka.KafkaSinkConfiguration
+import kotlin.test.assertEquals
+
+class StreamsTopicServiceTest {
+
+    private lateinit var db: GraphDatabaseService
+    private lateinit var streamsTopicService: StreamsTopicService
+
+    private lateinit var kafkaConfig: KafkaSinkConfiguration
+
+    @Before
+    fun setUp() {
+        db = TestGraphDatabaseFactory()
+                .newImpermanentDatabaseBuilder()
+                .newGraphDatabase()
+        kafkaConfig = KafkaSinkConfiguration(streamsSinkConfiguration = StreamsSinkConfiguration(topics = mapOf("shouldWriteCypherQuery" to "MERGE (n:Label {id: event.id})\n" +
```

Comment:
I wonder if we should have some "special" query tokens like `CREATE_NODE_FROM_TOPIC_EVENT` etc.
But that's perhaps a stupid idea and we should keep it explicit.

Reply:
I think we could leave this possibility to the end user, being the template configurable the user can inject fixed params in the query. Btw I can open an issue if you want